Document/0x06d-Testing-Data-Storage.md

```diff
@@ -24,45 +25,15 @@ Steps :
 
-#### White-box Testing
-
-When going through the source code it should be analyzed if native mechanisms that are offered by iOS are applied to the identified sensitive information. Ideally sensitive information should not be stored on the device at all. If there is a requirement to store sensitive information on the device itself, several functions/API calls are available to protect the data on IOS device by using for example the Keychain.
-
-#### Remediation
-
-If sensitive information (credentials, keys, PII, etc.) is needed locally on the device, several best practices are offered by iOS that should be used to store data securely instead of reinventing the wheel or leave it unencrypted on the device.
-
-The following is a list of best practice used for secure storage of certificates and keys and sensitve data in general:
-* For small amounts of sensitive data such as credentials or keys use the [Keychain Services](https://developer.apple.com/reference/security/1658642-keychain_services?language=objc) to securely store it locally on the device. Keychain data is protected using a class structure similar to the one used in file Data Protection. These classes have behaviors equivalent to file Data Protection classes, but use distinct keys and are part of APIs that are named differently. The the default behaviour is `kSecAttrAccessibleWhenUnlocked`. For more information have a look at the available modes [Keychain Item Accessibility](https://developer.apple.com/reference/security/1658642-keychain_services/1663541-keychain_item_accessibility_cons).
-* Cryptographic functions that have been self implemented to encrypt or decrypt local files should be avoided.  
-* Avoid insecure storage functions for sensitive information such as credentials and keys as illustrated in chapter OMTG-DATAST-001-2.   
-
-
-#### References
-
-* [Keychain Services Programming Guide](https://developer.apple.com/library/content/documentation/Security/Conceptual/keychainServConcepts/iPhoneTasks/iPhoneTasks.html)
-* [IOS Security Guide](https://www.apple.com/business/docs/iOS_Security_Guide.pdf)
-
-### Testing for Sensitive Data Disclosure in Local Storage
-
-#### Overview
-
-Storing data is essential for many mobile applications, for example in order to keep track of user settings or data a user might has keyed in that needs to stored locally or offline. Data can be stored persistently by a mobile application in various ways on each of the different operating systems. The following table shows those mechanisms that are available on the iOS platform:
-
-* CoreData/SQLite Databases
-* NSUserDefaults
-* Property List (Plist) files
-* Plain files
-
+-- TODO [Add content on Dynamic Testing of "Testing Local Data Storage "] --
```

Comment:
Can you please assign the issue to you? 

https://github.com/OWASP/owasp-mstg/issues/331